config/locales/server.en.yml

```diff
@@ -1931,6 +1931,10 @@ en:
     discord_secret: "Discord Client Secret Key Used for authenticating and enabling Discord related features on the site, such as Discord logins. This secret key corresponds to the Discord application created for the website, and is necessary for securely communicating with the Discord API."
     discord_trusted_guilds: 'Only allow members of these Discord guilds to log in via Discord. Use the numeric ID for the guild. For more information, check the instructions <a href="https://meta.discourse.org/t/configuring-discord-login-for-discourse/127129">here</a>. Leave blank to allow any guild.'
 
+    enable_linkedin_oidc_logins: "Enable LinkedIn authentication, requires linkedin_client_id and linkedin_client_secret."
```

Comment:
Since these other settings are required, we should make a site setting validator for `enable_linkedin_oidc_logins` to make sure they are filled in before enabling

Reply:
Added:

<img width="589" alt="Screenshot 2024-04-19 at 3 05 12 PM" src="https://github.com/discourse/discourse/assets/5259935/3cf895d3-fad4-408c-82e2-c4519585c055">

It doesn't protect from removing the credentials when auth is already enabled, but I think we fix that as part of our plan to describe setting interdependencies.

Reply:
Nice one, thanks!